docs/docs/03-quickstartheroku.md

```diff
@@ -6,3 +6,20 @@ title: Quick Start Heroku
 # Get up and running in seconds on Heroku.
 [<img src="https://www.herokucdn.com/deploy/button.svg">](https://heroku.com/deploy?template=https://github.com/Salesforce/refocus)
 
+## Running on Heroku
+- Setup Heroku account and toolbelt (follow this guide https://devcenter.heroku.com/articles/getting-started-with-nodejs#introduction)
```

Comment:
some but not all of these instructions are redundant if you clicked the heroku button--we should spell that out